sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/input_output.py

```diff
@@ -96,18 +96,15 @@ class Meta:
         allowed_values=SUPPORTED_PARAM_TYPES,
         required=True,
     )
-    # hide is_control and early_available in spec
+    # hide early_available in spec
     if is_private_preview_enabled():
-        is_control = fields.Bool()
         early_available = fields.Bool()
 
     def _serialize(self, obj, *, many: bool = False):
         """Override to add private preview hidden fields"""
         from azure.ai.ml.entities._job.pipeline._attr_dict import has_attr_safe  # pylint: disable=protected-access
 
         ret = super()._serialize(obj, many=many)  # pylint: disable=no-member
-        if has_attr_safe(obj, "is_control") and obj.is_control is not None and "is_control" not in ret:
-            ret["is_control"] = obj.is_control
```

Comment:
@wangchao1230 
Does it need to be added here _is_primitive_type=True?
I understand that it should not be necessary.